src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class Figure implements FigureInformation {
+    private static final Random random = new Random();
```

Comment:
All magic numbers in your code should be constants. Here, the number '100' used in the `getRandomInt` method should be declared as a constant.